pkg/kubelet/apis/config/types.go

```diff
@@ -445,6 +445,10 @@ type KubeletConfiguration struct {
 	// registerNode enables automatic registration with the apiserver.
 	// +optional
 	RegisterNode bool
+
+	// LocalStorageCapacityIsolation enables local ephemeral storage isolation. The default setting is true.
```

Comment:
It might be useful to give a short summary of what the feature does.

Reply:
added the document about feature and what case they need to set it false.